testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/util/OAuthClient.java

```diff
@@ -663,6 +723,32 @@ public CloseableHttpResponse doLogout(String refreshToken, String clientSecret,
         return client.execute(post);
     }
 
+    public CloseableHttpResponse doBackchannelLogout(String logoutTokon) {
```

Comment:
There is a typo here. It was meant to be `logoutToken` instead of `logoutTokon`.